src/main/kotlin/dev/rosewood/roseskyblock/island/IslandGroup.kt

```diff
@@ -2,7 +2,7 @@ package dev.rosewood.roseskyblock.island
 
 import dev.rosewood.roseskyblock.world.IslandWorld
 import dev.rosewood.roseskyblock.world.IslandWorldGroup
-import java.util.UUID
+import java.util.*
```

Comment:
A wildcard import for a single class seems overkill

Reply:
I likely imported something else from `java.util` there and so IntelliJ auto-corrected that, then I probably removed that thing I imported and IntelliJ didn't change it back because it saw nothing wrong with it.